ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml

```diff
@@ -3021,10 +3337,19 @@ If the client requests a text body whose native format is HTML, and the size of
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Unverified</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASAIRS_R1000977</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.18.2</ns1:Doc_Sect>
+		<ns1:Description>[In ContentType (Properties)] Protocol version 16.1 supports this [ContentType (Properties)] element.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
```

Comment:
adapter

Reply:
The similar RS below marked as Unverified in last RS update.

MS-ASAIRS_R1123 | 2.2.2.18.2 | [In ContentType   (Properties)] Protocol version 16.0 supports this [ContentType (Properties)]   element. |   | Protocol | Both | Normative | Unverified
-- | -- | -- | -- | -- | -- | -- | --